TAG

```diff
@@ -1 +1 @@
-2.7.100-tb
+2.7.101
```

Comment:
Why the dropping of the `-tb` postfix?  AFAICT this tag has always had `-tb` postfixed.  But to be completely honest, I don't recall why we have done that in the past or what the ramifications are of dropping it.